cloud-pubsub/pubsub-hpa.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: pubsub
+  namespace: default
```

Comment:
`namespace:`  is not necessary please avoid defaults

Reply:
done